### PR TITLE
Redesign voters guide

### LINKED ISSUE #42: Citizen options are not intuitive
The options that user has to choose from seem to be just content elements.
Can we make some changes?

1. add a radio button to each option
2. make the chosen option with yellow and leave the rest of options only with a white background and a gray border
3. can we add additional text above the options? exp: "Choose one of the options below:"

**now it's like this:**
image

**suggestion:**
image

### PR DESCRIPTION
**Pull request by idormenco**
closes #42

### CODE REVIEW

File: src/ClientApp/src/app/voters-guide/voters-guide.component.html [hunk 1]
```diff
@@ -1,20 +1,19 @@
 <ng-container *ngIf="data">
   <div class="container wrapper">
-    <p>{{ data.description }}
-    </p>
+    <p>{{ data.description }}</p>
+    <h3><u>Alege una din optiunile de mai jos:</u></h3>
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.html:4]:
```suggestion
    <h3><u>Alege una din opțiunile de mai jos:</u></h3>
```

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss [hunk 2]
```diff
@@ -1,49 +1,90 @@
...
-    .options-list {
-      .option {
-        width: 48%;
-        margin: 1%;
-      }
-    }
-  }
+	align-items: center;
+	margin-top: 50px;
+	p {
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.scss:4]: Please add a new line before each selector in this whole modifications block.

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss [hunk 3]
```diff
@@ -1,49 +1,90 @@
...
+			height: 100px;
+			margin: 5px;
+			font-weight: 600;
+			font-size: 16px;
+			line-height: 24px;
+			padding: 5px;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			border: #352245 solid 1px;
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.scss:26]:
```suggestion
			border: 1px solid #352245;
```

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss [hunk 4]
```diff
@@ -1,49 +1,90 @@
...
+				position: absolute;
+				top: 0;
+				right: 0;
+				transform: translateX(50%) translateY(-50%) rotate(45deg);
+				transition: all 0.2s ease-in-out;
+				opacity: 0;
+			}
+			.check-icon {
+				opacity: 0;
+				display: hidden;
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.scss:41]: `hidden` is an invalid value for `display` property. Use either `display: none` or  `visibility: hidden`.

In both cases, the `opacity: 0` above is redundant.

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss [hunk 5]
```diff
@@ -1,49 +1,90 @@
...
+				position: absolute;
+				top: 0;
+				right: 0;
+				transition: all 0.2s ease-in-out;
+				width: 25px;
+				height: 25px;
+				transform: translateX(10%) translateY(-10%);
+			}
+			&:hover {
+				border: #ffcc00 solid 2px;
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.scss:54]:
```suggestion
				border: 2px solid #ffcc00;
```

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss [hunk 6]
```diff
@@ -1,49 +1,90 @@
...
+				.check-icon-container {
+					opacity: 1;
+				}
+				.check-icon {
+					opacity: 1;
+				}
+			}
+			&:hover.is-selected {
+				border: #352245 solid 2px;
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.scss:63]:
```suggestion
				border: 2px solid #352245;
```

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss [hunk 7]
```diff
@@ -1,49 +1,90 @@
...
+			}
+		}
+		.option.is-selected {
+			background-color: #ffcc00;
+			border: #352245 solid 2px;
```
Comment (surdu) [inline src/ClientApp/src/app/voters-guide/voters-guide.component.scss:68]:
```suggestion
			border: 2px solid #352245;
```

**Review by surdu — CHANGES_REQUESTED** [review]

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss (hunk 4 above)
Comment (idormenco) [reply src/ClientApp/src/app/voters-guide/voters-guide.component.scss]: this property `opacity: 0` is afected by `transition: all` going from 1..0 and vice-versa so it should stay

**Review by catileptic — APPROVED** [review]

File: src/ClientApp/src/app/voters-guide/voters-guide.component.scss (hunk 4 above)
Comment (surdu) [reply src/ClientApp/src/app/voters-guide/voters-guide.component.scss]: `display: none` will hide the element completely. Changing it's opacity won't have any effect.

**Review by surdu — APPROVED** [review]

**Pull request merged**